scripts/README_write_gcr_to_parquet.md

```diff
@@ -0,0 +1,98 @@
+# Using write_gcr_to_parquet.py
+
+This README contains the updated instructions for using `write_gcr_to_parquet.py`.
+It is last updated by Yao-Yuan Mao on Jan 5th, 2021.
+
+## General environment setup
+
+### Step 1: Create a new working directory in scratch
+
+Most likely you would want to put the parquet files in scratch area first,
+and then copy them to the actual destination when everything is done.
+
+```bash
+cd $SCRATCH
+mkdir -p desc/a-new-working-dir
+cd desc/a-new-working-dir
+```
+
+### Step 2: Clone the needed repos
+
+To make sure that you are using the up-to-date scripts and catalog configs,
+you should clone the repos.
+If you want to test a specific branch, change the following accordingly.
+
+```bash
+git clone --depth=1 git@github.com:LSSTDESC/gcr-catalogs.git
+git clone --depth=1 git@github.com:LSSTDESC/DC2-production.git
+```
+
+### Step 3: Start up DESC Python environment
+
+```bash
+source /global/common/software/lsst/common/miniconda/setup_current_python.sh ""
+export PYTHONPATH="gcr-catalogs:$PYTHONPATH"
+```
+
+### Step 4 (optional but encouraged): Request an interactive session
+
+```bash
+salloc -N 1 -C haswell -q interactive -A m1727 -t 04:00:00
+```
+
+### Step 5: Generate Parquet files
+
+Generally speaking you can just run:
+
+```bash
```

Comment:
Saying "Generally speaking" and then giving a specific example with a real catalog can be confusing.  I suggest  replacing `cosmoDC2_v1.1.4` with a placeholder like `some-catalog`.   Or else change the text "generally speaking" to something that makes clear this is just an example.

Also since you are about to discuss changing the output filename prefix I would mention in this example what the output would be (by default, without `--output-filename` option)